awswrangler/distributed/ray/_core.py

```diff
@@ -24,27 +24,33 @@ def __init__(
     ):
```

Comment:
Also the log_level is set to INFO by default with no way for the user to change it when most of our loggers are set to debug

Reply:
I'd say `INFO` by default, but we should definitely allow to override it